diff --git a/src/lld/dpdk/Makefile.am b/src/lld/dpdk/Makefile.am
index 8cc21b3..38065dc 100644
--- a/src/lld/dpdk/Makefile.am
+++ b/src/lld/dpdk/Makefile.am
@@ -3,7 +3,7 @@ all:
 	$(shell ./apply_patch.sh > /dev/null 2>&1)
 	stat dpdk_src/build > /dev/null 2>&1 || \
         (cd dpdk_src && meson -Dprefix=$(prefix) -Dtests=false -Ddefault_library=shared \
-	 -Dexamples='' -Denable_kmods=false -Denable_docs=false -Denable_driver_sdk=false \
+	 -Dexamples='' -Denable_kmods=false -Denable_docs=false -Denable_driver_sdk=false -Dcpu_instruction_set=nehalem \
 	 -Ddisable_drivers=dma/*,raw/*,compress/*,crypto/*,event/*,baseband/* -Dbuildtype=release build && \
 	cd build && ninja -j4 && ninja install)
 	$(MAKE) -C infra install_dir=$(prefix)
