From 8a0b9f75724bba599374e5f10acef77064439e17 Mon Sep 17 00:00:00 2001
From: Yi Tseng <yi@opennetworking.org>
Date: Tue, 7 Jan 2020 16:51:18 +0800
Subject: [PATCH 2/2] patch

---
 bazel/generate_cc.bzl | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/bazel/generate_cc.bzl b/bazel/generate_cc.bzl
index 484959e..6c0cd51 100644
--- a/bazel/generate_cc.bzl
+++ b/bazel/generate_cc.bzl
@@ -32,6 +32,8 @@ def _strip_package_from_path(label_package, file):

 def _get_srcs_file_path(file):
     if not file.is_source and file.path.startswith(file.root.path):
+        if file.owner.workspace_root:
+            return file.path[len(file.root.path) + 1 + len(file.owner.workspace_root) + 1:]
         return file.path[len(file.root.path) + 1:]
     return file.path

--
2.24.1